docs/example/triggerDocs.vue

```diff
@@ -61,6 +61,28 @@
             <br>
             This is the same <trigger for="modal:trigger_id">trigger</trigger> as last one.
         </doc-code>
+        <br>
+        <h4>Trigger's <code>trigger</code> attribute (which defaults to <code>hover</code>) is independent of the target's.</h4>
```

Comment:
Would it be more appropriate to say that the `trigger` attribute of the target is being overridden/will be ignored, rather than just being independent?

Reply:
It is more appropriate to say *independent*. See updated example.